src/script/conversation/MessageRepository.ts

```diff
@@ -630,7 +643,7 @@ export class MessageRepository {
     return this.sendAndInjectGenericCoreMessage(payload, conversation);
   }
 
-  private isUserCancellationError(error: ConversationError): boolean {
+  private isUserCancellationError(error: any): boolean {
```

Comment:
why the decision to switch to an any type here? just curious

Reply:
That's a typescript constraint: Error are of type `unknown` and cannot be cast into a proper type. So you have  to cast it to `any` before you can do anything with it. 
 As of now the webapp compiles in non-strict mode, so TS doesn't complain about this, but in strict mode it doesn't allow an error to be cast.